typings/nui.ts

```diff
@@ -5,20 +5,21 @@ export type AccessTableUser = {
   stateId: string;
 };
 
+export interface Transaction {
+  date: string;
+  amount: number;
+  type: 'inbound' | 'outbound';
+  reason: string;
```

Comment:
I don't really like the name `reason` for this case, maybe a word like `description` instead?

I'm open to ideas and suggestions.

Reply:
I have in mind the reason of transactions but yeah can be changed to description or message like PEFCL

Reply:
Go for message